Jenkinsfile

```diff
@@ -83,9 +83,17 @@ pipeline {
 			}
                     }
                     steps {
+		        echo 'Cleaning targets'
+			sh 'cargo clean'
                         echo 'Beginning tests...'
                         sh 'cargo test --all'
                         echo 'Tests done!'
+			echo 'Install mdbook'
```

Comment:
Looks like the docker image [installs mdbook already](https://github.com/amethyst/builder/blob/9d8409287f885d0f8abe7807a7e31b4f2f5694e1/linux/Dockerfile#L32), so we don't have to.

Reply:
Thanks for checking this